db/fixtures/ae_datastore/ManageIQ/System/Event/EmsEvent/RHEVM.class/user_add_vm_template_finished_failure.yaml

```diff
@@ -7,6 +7,4 @@ object:
     name: USER_ADD_VM_TEMPLATE_FINISHED_FAILURE
     inherits: 
     description: 
-  fields:
-  - rel4:
-      value: "/System/event_handlers/event_action_policy?target=src_vm&policy_event=vm_discover&param="
+  fields: []
```

Comment:
Providers raise hundreds of events that we still capture and process.  Unless we are actually performing actions I do not see the point of making an exception for this event by having an empty reference here.  If a user needs to process this event they can define it in their own automate domain and wire the required actions.